### PR TITLE
Fixes nics format for lxc containers

### PR DESCRIPTION
**Pull request by Manisha15**

### CODE REVIEW

**Review by m-bucher — APPROVED** [review]
Looks good to me

**Review by sbernhard — APPROVED** [review]

**Review by sbernhard — APPROVED** [review]

**Comment by Manisha15** [comment]
@geemus Can you please review this? I cannot merge it unless someone with write access approves it.

**Review by geemus — APPROVED** [review]

**Pull request merged**
